scene/resources/primitive_meshes.cpp

```diff
@@ -1390,6 +1390,13 @@ void QuadMesh::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::VECTOR3, "center_offset"), "set_center_offset", "get_center_offset");
 }
 
+uint32_t QuadMesh::surface_get_format(int p_idx) const {
+	ERR_FAIL_INDEX_V(p_idx, 1, 0);
+
+	return RS::ARRAY_FORMAT_VERTEX | RS::ARRAY_FORMAT_NORMAL | RS::ARRAY_FORMAT_TANGENT | RS::ARRAY_FORMAT_TEX_UV;
+}
+
+
```

Comment:
This extra blank line is causing CI to fail. Just needs to be deleted

Reply:
Looks like I'll never have a clean PR without a few force pushes

Reply:
Haha! I don't know when the last time I made a PR without force pushing was.